### ISSUE TITLE
Followup Issues for Port to VORC

### ISSUE THREAD

**Issue by mabelzhang** [osrf/vrx-docker#20]
A meta-ticket of followup issues to #19. Applies to the `vorc` branch.

VORC essentials
- [ ] Need a new YAML file in `task_config` for the new gymkhana task
- [ ] In `task_config`, all trials > 0 coordinates need to be customized for VORC

Infrastructure
- [ ] Videos should be recorded from a window from server Docker `playback.launch`, as opposed to a window from the evaluator's host machine, which is likely on some arbitrary environment different from that in the server Docker.
- [ ] Fix https://github.com/osrf/vrx `vrx_gazebo` package setup to follow standard practices, so that the Python scripts for world generation can be used if `vrx_gazebo` is compiled with `catkin_make --install` as opposed to `catkin_make`. Currently they import from relative paths and cannot be used from outside the package.

Style
- [ ] This line in VRX competition evaluation https://github.com/osrf/vrx-docker/blob/878f499044e1c50d39188328ebc107b8a656eaf4/vrx_server/vrx-server/run_vrx_trial.sh#L65 should really use a separate launch file in `vrx_gazebo` in https://github.com/osrf/vrx `vrx_gazebo`, as opposed to piggybacking on `sandisland.launch`, which is an entry-level example file and doesn't need all the complexity associated with the `non_competition_mode` Boolean.
- [ ] The `non_competition_mode` Boolean should be renamed to avoid the double-negative, especially in the launch file, where it is used like "unless non_competition_mode"
- [ ] Look into why environment variable `GAZEBO_MODEL_PATH` needs to be set in the launch file with `non_competition_mode` Boolean. The parameter should really be `<export>`ed from `package.xml` and not need to be manually set.
- [ ] xacro files in `generated/task_generated/*/world_xacros/` can use better indentation

**Comment by crvogt** [comment]
Modifying `task_config`

**Comment by mabelzhang** [comment]
Thanks! By the way I forgot to say I'm doing the gymkhana task in bullet 1. I'm assuming you're taking on bullet 2? I'll open a PR for gymkhana and maybe you could help with trial numbers > 0 for that one as well......? :D

**Comment by crvogt** [comment]
Of course! Sorry, I just noticed that both bullets say task_config XD Yes, I'm working on bullet 2. Definitely, happy to do the gymkhana ones > 0.
